galaxy/templates/configmap-cvmfs-fix.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-configmap-cvmfs-fix
+  labels:
+    {{- include "galaxy.labels" . | nindent 4 }}
+data:
+  cvmfs-fix.sh: |
+    sleep 10;
+    while [ "$(kubectl get pods -n {{ .Release.Namespace }} -l 'app=cvmfscis' -l 'component=nodeplugin' -o custom-columns=STATUS:.status.phase --no-headers)" != "Running" ]; do
```

Comment:
Is the app label correct? Shouldn't it be `cvmfscsi` instead?
Also, will this work if there are multiple nodes and get pods returns multiple lines for running pods?

Reply:
Both good catches. Should be fixed now.